ova/Ova2Ovf.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+# Neova Health
```

Comment:
I would change the header of this file. The original file was from Neova Health, but we have modified a bit the script. Give the credits to Neova Health and said that it has been modified by Wazuh.

Reply:
Totally, I don't know when we changed the header but yes, give the credits to Neova Health and Wazuh modifications